contracts/FlowTransactionSchedulerUtils.cdc

```diff
@@ -284,27 +286,26 @@ access(all) contract FlowTransactionSchedulerUtils {
         /// @param timestamp: The timestamp of the transaction to remove
         /// @param handlerTypeIdentifier: The type identifier of the handler of the transaction to remove
         access(self) fun removeID(id: UInt64, timestamp: UFix64, handlerTypeIdentifier: String) {
+            pre {
+                self.handlerInfos.containsKey(handlerTypeIdentifier): "Invalid handler type identifier: Handler with type identifier \(handlerTypeIdentifier) not found in manager"
+                self.idsByTimestamp.containsKey(timestamp): "Invalid timestamp: Timestamp \(timestamp) not found in manager for transaction with ID \(id)"
+            }
```

Comment:
How does this impact the ability to remove an ID for a handler that's been removed?

Reply:
That isn't possible because a handler can't be removed unless all the transaction IDs for that handler have already been removed. I just put this in because before I was using `if let` to get the handler infos, but now that I am borrowing a reference, I needed to force unwrap, so I have these now just as a backup but these should never trigger